shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Core.hs

```diff
@@ -55,3 +61,6 @@ type SerialisableData t = (FromCBOR t, ToCBOR t)
 
 -- | Constraints for serialising from/to CBOR using 'Annotator'
 type AnnotatedData t = (FromCBOR (Annotator t), ToCBOR t)
+
+-- | Era STS map
+type family EraSTS (k :: Symbol) era :: Type
```

Comment:
I like the `Era-` prefix, we should use it for all of these type families :smile: 

Just a thought: what about `EraRule`?

Reply:
Yep, that's nicer :-)

Reply:
Can't we remove the `Era` part. In this way we avoid the repetition of eras: for instance `Core.EraRule "POOLREAP" era`. Why not simply `Core.Rule "POOLREAP" era`.

Reply:
I would drop the `Core.` prefix instead. I'm in favour of using the `Era-` prefix for all such type families so that we no longer have name conflicts such as `Core.Crypto` vs the `Crypto` class, `Core.TxBody` vs the various concrete `TxBody` types, ...